packages/EasyActivity/src/ActivityLogEntryFactory.php

```diff
@@ -66,4 +68,22 @@ public function create(string $action, object $object, array $changeSet): ?Activ
 
         return $logEntry;
     }
+
+    /**
+     * @param mixed[] $changeSet
+     *
+     * @return mixed[]
+     */
+    private function getClearedChangeSet(array $changeSet): array
+    {
+        return \array_filter($changeSet, static function (array $changeSetItem) {
+            if ($changeSetItem[0] instanceof DateTimeInterface && $changeSetItem[1] instanceof DateTimeInterface) {
+                $format = 'Y-m-d H:i:s.uP';
```

Comment:
Let's create `DATETIME_COMPARISON_FORMAT` for it.

Reply:
Can't we use Carbon to compare two dates instead of formatting them?

Reply:
How exactly do you suppose to compare them with Carbon?

Reply:
`Carbon::equalTo`, no?

Reply:
For `Carbon::equalTo` dates {date = "2021-12-20 06:38:59.446547", timezone_type = {int} 3, timezone = "UTC"} {date = "2021-12-20 13:38:59.446547", timezone_type = {int} 3, timezone = "Asia/Krasnoyarsk"}  are equal. It was decided to keep the changeSet if timezone was changed.

Reply:
> `Carbon::equalTo`, no?

For `Carbon::equalTo` dates {date = "2021-12-20 06:38:59.446547", timezone_type = {int} 3, timezone = "UTC"} {date = "2021-12-20 13:38:59.446547", timezone_type = {int} 3, timezone = "Asia/Krasnoyarsk"}  are equal. It was decided to keep the changeSet if timezone was changed.